src/app/api/gql/flows-dataset/fragments/fragment-flow-history-data.graphql

```diff
@@ -0,0 +1,65 @@
+fragment FlowHistoryData on FlowEvent {
+  __typename
+  ... on FlowEventAborted {
+    eventId
```

Comment:
`eventId` and `eventTime` are present in all events - can they be extracted 1 level above without duplication?

Reply:
Fixed